modules/store/testing/src/mock_state.ts

```diff
@@ -1,8 +1,10 @@
-import { Injectable } from '@angular/core';
+import { Injectable, Signal } from '@angular/core';
 import { BehaviorSubject } from 'rxjs';
 
 @Injectable()
 export class MockState<T> extends BehaviorSubject<T> {
+  state!: Signal<T | undefined>;
+
   constructor() {
     super(<T>{});
   }
```

Comment:
Should we also update `MockState`? 👀 

```suggestion
  /**
   * @internal
   */
  readonly state: Signal<T>;

  constructor() {
    super(<T>{});

    this.state = toSignal(this, { manualCleanup: true, requireSync: true });
  }
```